apiserver/provisioner/provisioner.go

```diff
@@ -129,6 +130,7 @@ func NewProvisionerAPI(st *state.State, resources facade.Resources, authorizer f
 		InstanceIdGetter:        common.NewInstanceIdGetter(st, getAuthFunc),
 		ToolsFinder:             common.NewToolsFinder(configGetter, st, urlGetter),
 		ToolsGetter:             common.NewToolsGetter(st, configGetter, st, urlGetter, getAuthOwner),
+		NetworkConfigAPI:        networkingcommon.NewNetworkConfigAPI(st, getCanModify),
```

Comment:
very nice to see the number of deleted lines exceed the number of added lines. :)